.github/workflows/slots-cleanup-weekly.yml

```
@@ -0,0 +1,89 @@
+name: Check Active PRs and Slots
```

name of workflow doesn't align with filename

```suggestion
name: Weekly app service slot cleanup
```